bin/runtime/src/lib.rs

```diff
@@ -670,6 +670,30 @@ impl pallet_contracts::Config for Runtime {
     type RelaxedMaxCodeLen = ConstU32<{ 256 * 1024 }>;
 }
 
+parameter_types! {
+    pub const BasicDeposit: Balance = 258 * DEPOSIT_PER_BYTE;
+    pub const FieldDeposit: Balance = 66 * DEPOSIT_PER_BYTE;
+    pub const SubAccountDeposit: Balance = 85 * DEPOSIT_PER_BYTE;
```

Comment:
it would be good to provide some explanation/calculations behind these values

Reply:
for now it should be sufficient

Reply:
done